snapcraft/extractors/appstream.py

```diff
@@ -42,7 +42,7 @@
     </xsl:copy>
 </xsl:template>
 
-<xsl:template match="p" xml:space="preserve">
```

Comment:
intentional?

Reply:
yes, the newer libxml works as intended and this makes all the spaces in this block count